src/main/kotlin/stringcalculator/core/Calculator.kt

```diff
@@ -0,0 +1,9 @@
+package stringcalculator.core
+
+object Calculator {
+    fun sum(numbers: List<Number>): Int {
+        val sum = numbers.reduce { number1, number2 -> number1 + number2 }
```

Comment:
메서드 참조를 이용하면 조금 더 깔끔하게 변경할 수 있을 것 같네요. 😃

```suggestion
        val sum = numbers.reduce(Number::plus)
```